rest/src/main/kotlin/route/Route.kt

```diff
@@ -366,7 +529,11 @@ sealed class Route<T>(
     object IntegrationId : Key("{integration.id}")
     object WebhookId : Key("{webhook.id}", true)
     object WebhookToken : Key("{webhook.token}")
-    object TemplateCode: Key("{template.code}")
+    object TemplateCode : Key("{template.code}")
+    object ApplicationId : Key("{application.id}")
+    object CommandId : Key("{command.id}")
+    object InteractionId : Key("interaction.id")
+    object InteractionToken : Key("{interaction.token}")
```

Comment:
I don't think it's documented anywhere, but are any of these 'major' keys? I assume they are.